backend/src/routes.js

```diff
@@ -24,6 +24,21 @@ router.get('/users/:id', async (req,res) => {
 });
 
 
+//route to get a single user
+router.get('/users/:id', async (req,res) => {
+    try{
+        let user = await Users.findById(req.params.id);
```

Comment:
Mike, I don't think this line of code is correct.
"Users" should be undefined here and Users.findById(..) probably will raise an exception.

The Schemas and Models have been modified to singular form, and to access them you should require the Schema.js
An example of this is line 10 of "index.js" and also a comment I have posted on slack - back-end channel.

Would you please run the server and verify the code is both running and the result is as expected?
I may be wrong in my assumption, in which case please forgive me and disregard this comment.

Best regards and have a great day